set.mm

```diff
@@ -179333,6 +179333,73 @@ other components such as a multiplicative operation for rings (and still
   remain a group per our definition).  Thus, every ring is also a group.  This
   allows theorems from more general structures (groups) to be reused for more
   specialized structures (rings) without having to reprove them.
+  Structures are common in mathematics, but in informal (natural language)
+  proofs the details are assumed in ways that we must make explicit.
+
+  There are many other ways to handle structures.  We chose this
+  "extensible structure" approach
+  because (1) this approach results in simpler notation than other
+  approaches we're aware of and (2) it is easier to do proofs with.
+  We cannot use an approach that uses "hidden" arguments; Metamath does
+  not support hidden arguments, and in any case we want nothing hidden.
+  It would be possible to use a categorical approach (e.g., something
+  vaguely similar to Lean's mathlib).
+  However, instances (the chain of proofs that an
+  X is a Y via a bunch of forgetful functors) can
+  cause serious performance problems for automated tooling,
+  and the resulting proofs would be painful to look at
+  directly (in the case of Lean, they are long past the level where
+  people would find it acceptable to look at them directly).
+  Metamath is working under much stricter conditions than this,
+  and it has still managed to achieve about the same level of flexibility.
+
+  To create a substructure of a given exensible structure, you can simply
+  use the multifunction restriction operator for extensible structures ` |`s `
```

Comment:
Warning, you have to double the ` within math formula embedded in a comment!
(same in other instances below)

Reply:
Fixed!